src/planscape/impacts/services.py

```diff
@@ -20,6 +20,7 @@
 ActionType = Type[TreatmentPrescriptionType]
 ProjectAreaType = Type[ProjectArea]
 TreatmentPrescriptionEntityType = Type[TreatmentPrescription]
+CloneResultType = Tuple[TreatmentPlanType, List[TreatmentPrescriptionEntityType]]
```

Comment:
Naming nit: if there's a chance we'll clone other things, could we name this something like TreatmentPlanCloneType ?